src/main/java/com/study/gamereviewrest/controller/GameController.java

```diff
@@ -0,0 +1,50 @@
+package com.study.gamereviewrest.controller;
+
+import com.study.gamereviewrest.exceptionHangler.GameNotFoundException;
+import com.study.gamereviewrest.model.Game;
+import com.study.gamereviewrest.model.Review;
+import com.study.gamereviewrest.repository.GameRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
```

Comment:
I see only one controller, there should be 2
1. game controller `/v1.0/games`
2. review controller `/v1.0/games/{id}/reviews`